internal/adapters/cloudformation/aws/elb/loadbalancer.go

```diff
@@ -33,7 +33,7 @@ func getListeners(lbr *parser.Resource, ctx parser.FileContext) (listeners []elb
 			listener := elb.Listener{
 				Metadata:       r.Metadata(),
 				Protocol:       r.GetStringProperty("Protocol", "HTTP"),
-				TLSPolicy:      r.GetStringProperty("SslPolicy", "ELBSecurityPolicy-2016-08"),
+				TLSPolicy:      r.GetStringProperty("SslPolicy", ""),
```

Comment:
Is there a failing test that we would need to update after this change? If not, we should add a test for it.

Reply:
No, there is no any test file for cloudformation adapter.

Reply:
OK can we add a simple unit test in that case? The fix is straight forward but having a unit test will guard us against regressions in the future.

Reply:
I did not write unit test for cloudformation file before, so i have no idea how to test it..

Reply:
OK looks like CloudFormation could be better tested. I'll open a separate issue for it as it seems there's no test in place at the moment. 

For your reference, we test the terraform equivalent as so: https://github.com/aquasecurity/defsec/blob/master/internal/adapters/terraform/aws/elb/adapt_test.go

Reply:
Issue opened here: https://github.com/aquasecurity/defsec/issues/1257